apicapi/apic_client.py

```diff
@@ -351,14 +351,14 @@ class ManagedObjectClass(object):
     prefix_to_mos = {
         (x.rn_fmt[:x.rn_fmt.find('-')] if '-' in x.rn_fmt else x.rn_fmt):
             _to_klass_name(y)
-        for y, x in supported_mos.items()
+        for y, x in list(supported_mos.items())
```

Comment:
In python3:
'dict.items()' has a type of `dict_items` and not `list`.
'dict.values()' has a type of `dict_values` and not `list`.
'dict.keys()' has a type of `dict_keys` and not `list`.

That's why for compatibility in python3 & python2 both, we need to type-cast them into list.
It might look redundant for py2, but very much needed for py3.

Reply:
In the link you provided, they actually have an example showing that we don't have to use list() to wrap around the value returned from invoking items(). Here is the example illustrated in that link,

for key, value in sammy.items():
    print(key, 'is the key for the value', value)

In that case, I suspect that we also don't need to use list() to wrap around values() and keys() API. Can you double-check?

Reply:
I agree when we are using for only iteration we don't need to wrap around with `list`. So here it might work. Thanks for pointing it out!